lib/MusicBrainz/DataStore/Redis.pm

```diff
@@ -48,6 +48,14 @@ sub get {
     return defined $value ? $self->_json->decode($value) : undef;
 }
 
+sub mget {
+    my ($self, @keys) = @_;
+
+    map {
+        defined $_ ? $self->_json->decode($_) : ()
```

Comment:
This should be `... : undef`, not `... : ()`. As it is, undefined values are completely dropped from the list; i.e. if `alert_mtime` and `notes_viewed` are `undef`, the value for `notes_updated` will end up in the position for `alert_mtime`.